app/experimenter/experiments/constants/nimbus.py

```diff
@@ -33,15 +33,18 @@ class NimbusTargetingConfig:
 )
 
 TARGETING_FIRST_RUN = NimbusTargetingConfig(
-    name="First start-up users (en)",
-    slug="first_run",
+    name="First start-up users (en) about:welcome",
+    slug="first_run_about_welcome",
     description=(
         "First start-up users (e.g. for about:welcome) with an en-* "
         "locale using the release channel."
     ),
     targeting=(
-        "localeLanguageCode == 'en' "
-        "&& (isFirstStartup || experiment.slug in activeExperiments)"
+        'localeLanguageCode == "en" '
+        "&& ((isFirstStartup "
+        "&& !('trailhead.firstrun.didSeeAboutWelcome'|preferenceValue)) "
+        "|| '{experiment.slug}' in activeExperiments) "
+        "&& 'app.shield.optoutstudies.enabled'|preferenceValue"
```

Comment:
i'm not sure what the requirements are for this/if this is generally needed

Reply:
its safe to asume these are needed for first run about:welcome experiments

Reply:
quick question, is it possible to further narrow targeting to windows 10 and en-US only?